orttraining/orttraining/core/graph/pipeline_transformer.cc

```diff
@@ -1135,9 +1278,11 @@ Status ApplyPipelinePartitionToMainGraph(
 
```

Comment:
Could you add comments to explain the status of `graph` after each step in this function? For example, from the name of this function `SplitGraph`, I'd expect the graph after this function only contains a sub-graph for a specific pipeline stage.

Reply:
`SplitGraph` only disconnect the graph based on cut_info. It splits edges but don't remove nodes. The sub-graph is created in the `GenerateSubgraph`. I can add some comments to make it more clear.

---
In reply to: [452411986](https://github.com/microsoft/onnxruntime/pull/4321#discussion_r452411986) [](ancestors = 452411986)